discovery/aws/ec2.go

```diff
@@ -342,6 +351,12 @@ func (d *EC2Discovery) refresh(ctx context.Context) ([]*targetgroup.Group, error
 								strings.Join(ipv6addrs, ec2LabelSeparator) +
 								ec2LabelSeparator)
 					}
+					if len(primaryipv6addrs) > 0 {
+						labels[ec2LabelPrimaryIPv6Addresses] = model.LabelValue(
+							ec2LabelSeparator +
+								strings.Join(primaryipv6addrs, ec2LabelSeparator) +
```

Comment:
Can there be `""` entries in the slice at this point?

Reply:
Yes, they can and it is intentional. I saw you wrote later that you read the description. Is this okay for you? I think the position can add some extra information which is useful in some cases.